sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -71,11 +71,19 @@ export class Notification extends RecordDataModel {
  */
 export enum NotificationMessageType {
   /**
-   * Notification Message type student file upload.
+   * Student uploaded a file into his account.
    */
   StudentFileUpload = 1,
   /**
-   * Notification  Message type ministry file upload.
+   * Ministry uploaded a file to the student account.
    */
   MinistryFileUpload = 2,
+  /**
+   * A new restriction was added to the student account.
+   * Possible scenarios:
+   * 1. Ministry places restriction on student account;
```

Comment:
`;`?.same in line 85

Reply:
Even not wrong, as we talked, I changed it because can be easier accepted 😉